src/models/atmosphere/FGWinds.h

```diff
@@ -394,8 +402,12 @@ class FGWinds : public FGModel {
   FGColumnVector3 vBurstGust;
   FGColumnVector3 vTurbulenceNED;
 
+  unsigned int RandomSeed;
   std::shared_ptr<RandomNumberGenerator> generator;
 
+  void SetRandomSeed(int sr);
+  int  GetRandomSeed(void) const { return RandomSeed; }
```

Comment:
You must account for the case where `SetRandomSeed` has not yet been called. In this case, `GetRandomSeed` must return the random seed from `FGFDMExec`.

Now that we have moved to C++17, this is a good use case for `optional<>`:
```suggestion
  std::optional<unsigned int> RandomSeed;
  std::shared_ptr<RandomNumberGenerator> generator;

  void SetRandomSeed(int sr);
  int  GetRandomSeed(void) const {
    if (RandomSeed)
      return *RandomSeed;
    else
      return FDMExec->SRand();
  }
```

Reply:
Okay, haven't used `optional<>` before, but will need to make `unsigned int FGFDMExec::SRand()` public since it's currently private so `FGWind` currently can't call it.